packages/db/src/loaders/commands/compile.ts

```diff
@@ -72,6 +57,7 @@ export function* generateCompileLoad(
   //
   // again going one compilation at a time (for impl. convenience; HACK)
   // (@cds-amal reminds that "premature optimization is the root of all evil")
+  let contractsByCompilation = [];
```

Comment:
We had talked about not having this here, and instead using the contracts that are inside the compilations. But I sorta like doing it this way because then we are sure to only return those contracts that have been loaded, right? If I understand this correctly.